src/components/topbar/TopMenubar.vue

```diff
@@ -94,13 +78,20 @@ eventBus.on((event: string, payload: any) => {
   }
 })
 
-onMounted(() => {
-  if (isElectron()) {
-    electronAPI().changeTheme({
-      height: topMenuRef.value.getBoundingClientRect().height
-    })
-  }
-})
+/** Height of titlebar on desktop. */
```

Comment:
I don' think the resize observer is necessary after https://github.com/Comfy-Org/ComfyUI_frontend/pull/2209 as the topbar menu height is now fixed.